src/fft.jl

```diff
@@ -0,0 +1,190 @@
+using Primes
+
+
+ωcos(N::Integer, i::Integer) = cospi(-2i//N)
+ωsin(N::Integer, i::Integer) = sinpi(-2i//N)
+ωcos(N, i) = cospi(-2i/N)
+ωsin(N, i) = sinpi(-2i/N)
+
+ω(N, i) = ωcos(N, i) + im*ωsin(N, i)
```

Comment:
Does the generated code take advantage of the many trig factors that are +/-1 or +/-i?